.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## Description
```

Comment:
@yo1995 - I'm not sure why this file is in this PR. Does it need to be?

Reply:
It doesn't need to be in this PR. The template file only needs to be on the main branch. The commit history in this PR is pretty messy so I cannot tell where it comes from, but probably at some point Chris branched off the `main` branch, or did a rebase, etc.

I'd probably squash merge this PR when it is approved, to avoid messing up the timeline.